Fakebook.Posts/Fakebook.Posts.RestApi/Controllers/PostsController.cs

```diff
@@ -274,11 +318,8 @@ public async Task<ActionResult> UploadPicture(IFormFile file, string userId)
         /// A user's newsfeed contains the three most recent posts
         /// of the user and the users they follow.
         /// </summary>
-        /// <returns>
-        /// Ok responce with the top 3
-        /// </returns>
-        // Route: api/newsfeed
-
+        /// <returns>An IActionResult containing either a:
+        /// 200 OK on success</returns>
```

Comment:
this doesn't make sense with one option

Reply:
Fixed.  That being said, I'm still not sure how I feel about that method being in our service in general.  Is it really the Post service's job to store the profile picture of the user who posted that post?  Not really sure, seems strange

Reply:
@ColeSamuelsen23 it took me a minute to figure out what part of the code you're talking about, since it's the method above, not the one this comment was about.
yeah, the profile service should probably be handling profile picture upload.